src/controller/CHIPDeviceController.cpp

```diff
@@ -1146,14 +1148,69 @@ void DeviceCommissioner::OnDeviceAttestationInformationVerification(
         }
     }
     else
+    {
+        {
+            ChipLogProgress(Controller, "Successfully validated 'Attestation Information' command received from the device.");
+            commissioner->CommissioningStageComplete(CHIP_NO_ERROR);
+        }
+    }
+}
+
+void DeviceCommissioner::OnDACChainRevocationStatusVerification(
+    void * context, const Credentials::DeviceAttestationVerifier::AttestationInfo & info, AttestationVerificationResult result)
```

Comment:
Why is this doing an AttestationVerificationResult?  Shouldn't this just return a "revoked or not" boolean, more or less?  Or some other enum?  It doesn't make sense to have most AttestationVerificationResult values for a revocation check?

Reply:
AttestationVerificationResult already contains DAC, PAI and PAA revocation status.